lib/TabTracker.js

```diff
@@ -76,7 +79,7 @@ TabTracker.prototype = {
 
   logActivate: function(tab) {
     if (this.isActivityStreamsURL(tab.url)) {
-      ss.storage.tabData[tab.url][tab.id].activations.push({startTime: Date.now()});
+      this._tabData[tab.url][tab.id].activations.push({startTime: Date.now()});
```

Comment:
yes, I just separated them so I can have something review-able by eod and I'll get to the structure change in another PR.